migrator/clone/postgres/db_clone_manager_impl_test.go

```diff
@@ -232,36 +225,37 @@ func (s *PostgresCloneManagerSuite) TestGetCloneMigrateRocks() {
 		LastPersisted: time.Now(),
```

Comment:
It looks like the LastPersisted field in the MigrationVersion is not read anymore.
Would it make sense to remove it from the struct definition and stop setting it ?